graph/telemetry/istio/appender/ambient.go

```diff
@@ -0,0 +1,71 @@
+package appender
+
+import (
+	"context"
+	"strings"
+
+	"github.com/kiali/kiali/business"
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/graph"
+	"github.com/kiali/kiali/log"
+)
+
+const AmbientAppenderName = "ambient"
+const WaypointSuffix = "-istio-waypoint"
+
+type AmbientAppender struct {
+	Waypoints bool
+}
+
+// Name implements Appender
+func (a AmbientAppender) Name() string {
+	return AmbientAppenderName
+}
+
+// IsFinalizer implements Appender
+func (a AmbientAppender) IsFinalizer() bool {
+	return true
```

Comment:
I think it's currently OK for this to be a Finalizer appender, and run only one time at the end of graph generation.  But we may find at some point that this be run per-namespace, depending on whether we add more ambient logic here, want to eliminate running against non-ambient namespaces, etc.

Reply:
Should we add any comment and leave it as it is now?

Reply:
We can leave it as-is.